StabilityPackage.wl

```diff
@@ -350,12 +350,12 @@ Export[ToString[name]<>".h5",  {"/unique_elements/r_indicies"->{"Data"-> DeleteD
 "/grid_elements/k"-> {"Data"-> outevs[[1,All,3]],"Attributes"-> {"Units"-> "Ergs"}},
 "/grid_elements/evs_Re"-> {"Data"-> Re[outevs[[1,All,4,1]]],"Attributes"-> {"Units"-> "Ergs"}},
 "/grid_elements/evs_Im"-> {"Data"-> Im[outevs[[1,All,4,1]]],"Attributes"-> {"Units"-> "Ergs"}},
-"/grid_elements/evecs_nu_Re"-> {"Data"-> Re[Normalize[outevs[[1,All,4,2,1;;10]]]],"Attributes"-> {"Normalized"}},
-"/grid_elements/evecs_nu_Im"-> {"Data"-> Im[Normalize[outevs[[1,All,4,2,1;;10]]]],"Attributes"-> {"Normalized"}},
-"/grid_elements/evecs_nubar_Re"-> {"Data"-> Re[Normalize[outevs[[1,All,4,2,11;;20]]]],"Attributes"-> {"Normalized"}},
-"/grid_elements/evecs_nubar_Im"-> {"Data"-> Im[Normalize[outevs[[1,All,4,2,11;;20]]]],"Attributes"-> {"Normalized"}},
+"/grid_elements/evecs_nu_Re"-> {"Data"-> Re[outevs[[1,All,4,2]][[All,All,1;;10]]],"Attributes"-> {"Norm?"-> "Unnormalized"}},

```

Comment:
This really requires a comment explaining what all of the indices in outevs are

Reply:
Sure. I was sort of thinking by identifying each element by name, that this was unnecessary since outevs isn't used, but it would be helpful if changes need to be made in the future.

Reply:
I think a simple comment can also go a long way. Automation is important, but there is a (entirely subjective) line between automation and over-engineering.

Reply:
Can do. I see your point about over engineering.  These indices are one of the reasons I was so excited about the HDF format!